pyproject.toml

```diff
@@ -46,9 +54,8 @@ Repository = "https://github.com/beetbox/mediafile"
 requires = ["poetry-core"]
```

Comment:
Seems a bit strange to me to still use the poetry build system. We should pick another one.